internal/payloads/error_payload.go

```diff
@@ -108,27 +106,13 @@ func NewNotFoundError(ctx context.Context, err error) *ResponseError {
 	}
 }
 
-func NewInitializeDAOError(ctx context.Context, message string, err error) *ResponseError {
-	msg := "initialize dao error"
-	if logger := ctxval.Logger(ctx); logger != nil {
-		logger.Error().Msg(msg)
-	}
-	return &ResponseError{
-		HTTPStatusCode: 500,
-		Message:        msg,
-		TraceId:        ctxval.TraceId(ctx),
-		Error:          err,
-	}
-}
-
```

Comment:
don't we need that error?

Reply:
Yep, that is correct. No DAO init errors, yay!!! :-)

Reply:
great! removing it is :)